lib/player.js

```diff
@@ -4371,6 +4467,102 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     return shaka.util.StreamUtils.textStreamToTrack(stream);
   }
 
+  /**
+   * Adds the given chapters track to the loaded manifest.  <code>load()</code>
```

Comment:
Does DASH offer a way to include a chapters track reference in the manifest?  Or will it always have to be side-loaded for DASH content?

Does HLS offer a way to include a chapters track in a playlist?  If so, does Apple's native HLS expose them through `video.textTracks`?

Reply:
As far as I know, both DASH and HLS do not offer this functionality and it is always side-loaded